deployments/fargate/README.md

```diff
@@ -0,0 +1,137 @@
+# AWS Fargate Deployment
+Familiarity with AWS Fargate (Fargate) is assumed. Consult the 
+[User Guide for AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/userguide/what-is-fargate.html)
+for further reading.
+
+Unless stated otherwise, it is assumed that the
+[Splunk OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector)
+(Collector) is deployed to an ECS Task in a **sidecar** container alongside monitored
+applications.
+
+Applies to Collector release v0.30.0 and above which corresponds to image tag 0.30.0 and above
+in the image repository [here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
+
+Overview of the container definition for the Collector:
+- Set environment variable `SPLUNK_CONFIG` to the configuration file path. Set `SPLUNK_CONFIG`
+  to the default configuration file path to enable default configuration.
+- Use extension `ecs_observer` in your custom configuration to discover targets.
```

Comment:
The default config has this right? Is there a link to additional documentation? If not, why is this included?

Reply:
No, the default config does not have ecs_observer. Mainly because we don't know the task name or rather arn ahead of time.